net.longturn.freeciv21.yaml

```diff
@@ -0,0 +1,74 @@
+##########
+# Freeciv21 - net.longturn.freeciv21.yaml
+#
+# SPDX-License-Identifier: GPL-3.0-or-later
+# SPDX-FileCopyrightText: James Robertson <jwrober@gmail.com>
+#
+# Build Steps:
+#   - (once) sudo apt install flatpak-builder
+#   - (once) sudo flatpak remote-add --if-not-exists flathub https://flathub.org/repo/flathub.flatpakrepo
+#   - flatpak-builder --install-deps-from=flathub --force-clean build/flatpak dist/net.longturn.freeciv21.yaml
+#   - flatpak-builder --user --install --force-clean build/flatpak dist/net.longturn.freeciv21.yaml
+#   - flatpak run net.longturn.freeciv21
+#   - flatpak remove net.longturn.freeciv21
+#
+##########
+
+app-id: net.longturn.freeciv21
+runtime: org.kde.Platform
+runtime-version: '5.15-22.08'
+sdk: org.kde.Sdk
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+  - --share=network
+  - --socket=pulseaudio
+  - --filesystem=home
+  - --filesystem=~/.local/share/freeciv21
```

Comment:
these broad filesystem permssions are redundant.

Also are you sure `~/.local/share/freeciv21` is used inside the sandbox? Standard Qt API do respect XDG.

Reply:
Let me try on my local. I may not need it.

Reply:
ok works fine on my local without. will remove